examples/simple/setup.py

```diff
@@ -0,0 +1,46 @@
+import os, setuptools
+from setuptools import find_packages
+
+VERSION = '0.0.1'
+
+def get_data_files():
+    """Get the data files for the package.
+    """
+    data_files = [
+        ('etc/jupyter/jupyter_server_config.d', ['etc/jupyter/jupyter_server_config.d/simple_ext.json']),
+    ]
+    def add_data_files(path):
+        for (dirpath, dirnames, filenames) in os.walk(path):
+            if filenames:
+                data_files.append((dirpath, [os.path.join(dirpath, filename) for filename in filenames]))
+    # Add all the templates
+    add_data_files('simple_ext1/static')
+    add_data_files('simple_ext1/templates')
+    add_data_files('simple_ext2/static')
+    add_data_files('simple_ext2/templates')
+    return data_files
+
+setuptools.setup(
+    name = 'simple_ext',
+    version = VERSION,
+    description = 'Jupyter Simple Extension',
+    long_description = open('README.md').read(),
+    packages = find_packages(),
+    python_requires = '>=3.5',
+    install_requires = [
+        'jupyter_server==0.1.1',
```

Comment:
I would drop this hardcoded dependency, as this example actually breaks with this version.

Reply:
Done. Thx, Eric